orchestrate.py

```diff
@@ -226,7 +226,6 @@ def main():
     )
 
     spawner = dockworker.DockerSpawner(docker_host,
-                                       version=opts.docker_version,
```

Comment:
Hm. Yes. Will update in 22 minutes.
On Aug 27, 2015 7:52 AM, "Kyle Kelley" <notifications@github.com> wrote:

> In orchestrate.py
> <https://github.com/jupyter/tmpnb/pull/159#discussion_r38104210>:
>
> > @@ -226,7 +226,6 @@ def main():
> >      )
> >
> >      spawner = dockworker.DockerSpawner(docker_host,
> > -                                       version=opts.docker_version,
>
> I think it's ok if they still want to specify it, since that's part of our
> current command line options. It's just the default that should be auto.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/tmpnb/pull/159/files#r38104210>.
>